UsefulProteomicsDatabases/ProteinDbLoader.cs

```diff
@@ -84,12 +84,31 @@ public static class ProteinDbLoader
                             Protein newProtein = block.ParseEndElement(xml, modTypesToExclude, unknownModifications, isContaminant, proteinDbLocation);
                             if (newProtein != null)
                             {
-                                targets.Add(newProtein);
+                                potentialTargets.Add(newProtein);
                             }
                         }
                     }
                 }
             }
+            HashSet<string> unique_accessions = new HashSet<string>();
```

Comment:
Can you include a comment that this is for isoform numbering? You could probably refactor from foreach() to for() and remove the "potentialTargets" variable, too.

Reply:
Will do.

Reply:
With fix in variant application lines 93-112 can be removed. The main issue was in line 116 squashing together isoforms.